firstvoices/backend/search/documents/dictionary_entry_document.py

```diff
@@ -15,6 +15,9 @@ class DictionaryEntryDocument(BaseDocument):
     type = Keyword()
     custom_order = Keyword()
     categories = Keyword()
+    hasAudio = Keyword()
```

Comment:
Just wondering that should we use the Boolean field type for these attributes ?
https://www.elastic.co/guide/en/elasticsearch/reference/current/boolean.html

Any specific reason for using the keyword type ?
From my understanding, using Boolean is also more space and query efficient compared to keyword. I have also used boolean type for the games and kids flag for reference.

Reply:
+1 for bool here

Reply:
Good catch. Thanks!